### ISSUE TITLE
Take scoopable stars in account

### ISSUE THREAD

**Issue by Luzifer** [Luzifer/ed-fast-travel#1]
In order to have routes which does not drain the fuel because of 5 consecutive T-Tauri (or other unscoopable) stars take in account whether stars are scoopable.

**Solution 1:** Let every stop on the route be scoopable
- Not applicable when calculating routes with stops every 15Ly or so because there are regions without any scoopable stars which are broader than 15Ly
- Hard when calculating routes with small jump range because the route gets way more complex

**Solution 2:** Let every `N`th stop on the route be scoopable
- Would work for routes with small stop-distances as there still may be `N-1` stops unscoopable

In general it's not that useful when calculating with high distances because you definitely need to scoop more often than every 500Ly (except for ships built as flying fuel tanks)…

**Comment by Luzifer** [comment]
Currently not possible due to only ~13000 registered stars in EDDB. This will change when E:D 2.2 is released and the EDMC provides bodies data from the journal.

**Comment by bensleveritt** [comment]
You are doing the lord's work. Please continue!

**Comment by LucaBastelt** [comment]
Now that 2.2 is released, whats the status? :D

**Comment by Luzifer** [comment]
I'd love to have better news yet but currently the state is that we have around 3.5m star systems and only around 55k bodies in EDDB… It's already visible while doing the routing that we are slowly getting more knowledge about scoopable stars:

![image](https://cloud.githubusercontent.com/assets/20142/19864392/58268784-9f98-11e6-87e2-aca77f03538c.png)

Though it's not yet possible to build a route only (or mostly) containing scoopable star systems… I definitely have this on my roadmap and I'd love to have it implemented but we need to wait some more time until we have enough knowledge of bodies in EDDB for this…

If you're actively using EDMC you can check the settings and look for the option to send body data to EDDB. If you turn that option on and do scans of stars with a Detailed Surface Scanner you'll help filling the body database. That's exactly the information I'm currently using for displaying scoopable star systems and will be using for the routing mentioned in this issue…

So long story short: It's not yet possible but still on my todo-list… Please be patient a little longer… 😉
